docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+# Your project could have own hostname & database by edit lines marked with double sharp (##)
```

Comment:
I don't think this comment applies since I don't see any lines beginning with `##`.  In general, within this project, it's pretty rare to include comments.  I would suggest removing this comment and the others on this file.

Also, please remove the extra new lines.